### ISSUE TITLE
Sweep: Gaussian+Global+Cross Entropy, sweeping receptive field width

### ISSUE THREAD

**Issue by RylanSchaeffer** [RylanSchaeffer/FieteLab-No-Free-Lunch#1]
Experiment: Train various models on Gaussian receptive fields with global normalization and cross entropy loss, sweeping the variance of the Gaussians

W&B page: https://wandb.ai/rylan/mec-hpc-investigations/sweeps/xqyfdt1v

Results:

- Most models converged in ~10 epochs (250 grad steps per epoch)
![image](https://user-images.githubusercontent.com/8942987/160944077-a87d79e3-aa57-4dc3-98a5-a84028eada9f.png)

- Position decoding error also fell on the same timescale
![image](https://user-images.githubusercontent.com/8942987/160944181-5da0cadb-05c6-4222-894f-00309295a432.png)

- Barring a small number of exceptions, most d=60 grid cells remained under 1.0
![image](https://user-images.githubusercontent.com/8942987/160944250-5853f9dd-2a8c-47a0-844c-4e4b7db2af09.png)

- The run with the highest d=60 grid score (with value 1.44) does not look grid like, and no other grid cells are observed. W&B run: https://wandb.ai/rylan/mec-hpc-investigations/runs/ykssp3rk?workspace=user-rylan
![image](https://user-images.githubusercontent.com/8942987/160944355-0c4195a1-1b2f-49f4-98ba-42a277bb21f4.png)
